dev/cnf/gradle/scripts/fat.gradle

```diff
@@ -324,6 +331,69 @@ assemble {
   }
 }
 
+ext.gitClone = { proj, branch = 'NOT_SET', folder = '', site = 'git@github.com', org = 'eclipse' ->
+
+  // We need to know which MP project at least
+  if( proj == null || proj.length() == 0 ){
+    project.logger.lifecycle("The first parameter to task clone is the project name and cannot be defaulted");
+    return
+  }
+
+  // Default the branch to master but allow -Dbranch= or a parameter coming in
+  if( "NOT_SET".equals( branch ) ){
+    String branchProp = System.getProperty('branch')
+    if( branchProp != null && branchProp.length() > 0 ){
+       branch = branchProp
+    }else{
+       branch = "master"
+    }
+  }
+  
+  // Default the destination folder to the project name
+  if( folder == null || folder.length() == 0 ){
+    folder = proj
+  }
+
+  //Staging area parent folder to enable copying all subfolders.
+  File gitReposDir = new File(project.buildDir, 'gitRepos')
+
+  // If running locally use git directly
+  if( System.properties['fat.test.localrun'] ){
+
+    File repoDir = new File(gitReposDir, folder)
+    String into = repoDir.getAbsolutePath()
+     
+    String url = " " + site + ":" + org + "/" + proj + ".git "
+    String cmd = "git clone --depth 1 --single-branch --branch " + branch + " " + url + " " + into
+      
+    delete into
+    mkdir into
+    
+    def proc = cmd.execute()
+    proc.in.eachLine { line -> println cmd + ':\n' + line }
+    proc.err.eachLine { line -> println cmd + ':\n' + line }
+    proc.waitFor()
+    
+  } else {
+    // Make use of the EBC clone mechanism
+    project.logger.lifecycle('ebc clone')
+    ebcClone(url) 
+    
+    //EBC clones in dev/../../../<repo>
+    File ebcCloneDir = new File( project, "./../../../" )
+    copy {
+      from ebcCloneDir
+      include proj
+      into gitReposDir
+    }
+  }
+  
+}
+
+def ebcClone(String url, String branch, String repoPath){
+  project.logger.lifecycle('ERROR: Needs pre-req of https://github.ibm.com/APTribe/CSI-Squad-Backlog/issues/372 to implement git clone of' + url)
```

Comment:
we cannot have dependencies on IBM-specific stuff in OpenLiberty, including github.ibm.com

Reply:
Hmmm - this is useful. ebcClone is only called when we are building non 'locally' (in the build cloud) I will make a change to make the code not reference IBM directly.

Reply:
Line removed.